package.json

```diff
@@ -235,6 +235,7 @@
     "react-loadable": "5.5.0",
     "react-paginate": "6.3.2",
     "react-popper": "0.7.4",
+    "react-progressbar-semicircle": "^1.2.1",
```

Comment:
was this semi circle progress bar approved by arunima? I don't really understand the point in using it versus a traditional progress bar

Reply:
It is part of v2 as well as v3 designs in Figma and no concerns were raised against progressbar

Reply:
we have replaced the circular progress bar with the traditional linear progress bar